src/loginUser/index.ts

```diff
@@ -0,0 +1,36 @@
+import { APIGatewayProxyHandler } from "aws-lambda";
+import { loginUser } from "../services/cognito";
+import { CognitoIdentityServiceProvider } from "aws-sdk";
+
+export const handler: APIGatewayProxyHandler = async (event) => {
+  const data = JSON.parse(event.body || "{}");
+
+  const { email, password } = data;
+
+  try {
+    const params: CognitoIdentityServiceProvider.InitiateAuthRequest = {
+      AuthFlow: "USER_PASSWORD_AUTH",
+      ClientId: process.env.APP_CLIENT_ID!,
+      AuthParameters: {
+        USERNAME: email,
+        PASSWORD: password,
+      },
+    };
+
+    const loginResponse = await loginUser(params);
+
+    return {
+      statusCode: 200,
+      body: JSON.stringify({
+        message: "Login successful!",
+        idToken: loginResponse.AuthenticationResult?.IdToken,
```

Comment:
i think you will need the access token for authentication

Reply:
yes, I changed for access token and not IdToken